CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+1.0.3
```

Comment:
I do not believe we ever released a 1.0.3 and these updates appear to be for 2.0.0 and this should be updated.